caiman/base/movies.py

```diff
@@ -108,6 +109,7 @@ def __new__(cls, input_arr, **kwargs):
 
     def motion_correction_online(self, max_shift_w=25, max_shift_h=25, init_frames_template=100,
                                  show_movie=False, bilateral_blur=False, template=None, min_count=1000):
+        # FIXME: This does not match the signature in motion_correction.py:motion_correct_online
```

Comment:
Almost certainly a bug

Reply:
Not sure I understand. Where is the bug?

Reply:
Sorry, I marked the wrong line; should be the line below where it calls motion_correct_online (but doesn't call it correctly)

Reply:
can you specify what the error is? Seems to me poor coding that just replicates a function rather than a bug

Reply:
The positional parameters to motion_correct_online() are:
movie_iterable, add_to_movie
(followed by a number of named arguments)

the call below only provides one positional parameter (the self object).